task3/Pasichnichenko_Task3.py

```diff
@@ -0,0 +1,19 @@
+from math import sin, atan
+
+# Multi-paradigm programming languages Lab2 Illia Pasichnichenko IKM-221a
+print('Multi-paradigm programming languages Lab2')
+print('Illia Pasichnichenko IKM-221a')
+
+a = float(input("Enter a value: "))
+while a == 0:
+    a = float(input('a cannot be equal to 0, enter valid number'))
```

Comment:
you have some logic duplication. You have **a = float(input** twice. Rewrite the code to avoid it